src/e2e-selectors/resolver.ts

```diff
@@ -0,0 +1,47 @@
+const semver = require('semver');
+import { E2ESelectors, VersionedSelectors } from './types';
+const processSelectors = (
```

Comment:
This is basically copied from grafana/e2e. Should be possible to clean this up and add better typings so we can stop ignoring ts errors.

Reply:
Sounds good and agree 👍🏻